templates/solr.in.sh.j2

```diff
@@ -68,15 +68,15 @@ SOLR_PORT="{{ solr_port }}"
 ZK_HOST="{{zk_host}}"
 {% endif %}
 
-{% if solr_auth_type is defined and solr_version is version('7.0.0', '>=') %}
+{% if solr_auth_configure and solr_auth_type is defined and solr_version is version('7.0.0', '>=') %}
 SOLR_AUTH_TYPE="{{ solr_auth_type }}"
 {% endif %}
 
-{% if solr_authentication_opts is defined and solr_version is version('7.0.0', '>=') %}
+{% if solr_auth_configure and solr_authentication_opts is defined and solr_version is version('7.0.0', '>=') %}
 SOLR_AUTHENTICATION_OPTS="{{ solr_authentication_opts }}"
 {% endif %}
 
-{% if solr_ssl_key_store is defined %}
+{% if solr_ssl_configure and solr_ssl_key_store is defined %}
```

Comment:
still need `solr_ssl_key_store is defined`?
solr_ssl_key_store is defined in vars, so it always be True